elephas/worker.py

```diff
@@ -42,11 +43,14 @@ def train(self, data_iterator):
 
         weights_before_training = self.model.get_weights()
         if x_train.shape[0] > self.train_config.get('batch_size'):
-            self.model.fit(x_train, y_train, **self.train_config)
+            history = self.model.fit(x_train, y_train, **self.train_config)
         weights_after_training = self.model.get_weights()
         deltas = subtract_params(
             weights_before_training, weights_after_training)
-        yield deltas
+        if history is not None:
```

Comment:
This is a second feature. My teammate wants to plot train_loss vs val_loss over time to see if the model over predicts, so I piped the history.history (i.e. a List[Dict]) from individual worker and let the driver collect all of them together with the ElephaseTransformer